win32ss/user/winsrv/usersrv/lang/pl-PL.rc

```diff
@@ -34,16 +34,16 @@ END
 
 STRINGTABLE
 BEGIN
-    IDS_SEVERITY_SUCCESS        "Success"
-    IDS_SEVERITY_INFORMATIONAL  "System Information"
-    IDS_SEVERITY_WARNING        "System Warning"
-    IDS_SEVERITY_ERROR          "System Error"
+    IDS_SEVERITY_SUCCESS        "Skukces"
+    IDS_SEVERITY_INFORMATIONAL  "Informacja"
+    IDS_SEVERITY_WARNING        "Uwaga"
+    IDS_SEVERITY_ERROR          "Błąd"
 END
 
 STRINGTABLE
 BEGIN
-    IDS_SYSTEM_PROCESS          "System Process"
-    IDS_UNKNOWN_SOFTWARE_EXCEPT "unknown software exception"
-    IDS_OK_TERMINATE_PROGRAM    "Click on OK to terminate the program."
-    IDS_CANCEL_DEBUG_PROGRAM    "Click on CANCEL to debug the program."
+    IDS_SYSTEM_PROCESS          "Proces"
```

Comment:
How do you convey the idea that this process is not any process but the "System" (aka. kernel-mode) process?
Note that the task manager translates this as "Proces bezczynności systemu" (for "System Idle Process").

Reply:
I think it should be "Proces systemowy" then.